python/ray/serve/_private/config.py

```diff
@@ -48,6 +49,33 @@ def _needs_pickle(deployment_language: DeploymentLanguage, is_cross_language: bo
         return False
 
 
+def _proto_to_dict(proto: Message) -> Dict:
```

Comment:
Nice this looks pretty simple. Add some basic tests for it?

Reply:
There are already existing config tests in [test_config.py](https://github.com/ray-project/ray/blob/46aac4db19c82d4fdb2f7f99c83c3bdb3a6ab051/python/ray/serve/tests/unit/test_config.py) so I didn't bother to write new ones. But add some more unit tests for calling `_proto_to_dict ` directly in a bit